tests/cover/test_conjecture_minimizer.py

```diff
@@ -17,15 +17,18 @@
 
 from __future__ import division, print_function, absolute_import
 
+from random import Random
```

Comment:
There isn't per se, but it's right there in the signature of function that it takes a Random object as an argument.